docs/howto/connection/sqoop.rst

```diff
@@ -0,0 +1,43 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Can you rebase the PR on latest master please -- we have changed the docs a bit for 2.0.

All the files in this PR need to be moved to https://github.com/apache/airflow/blob/master/docs/apache-airflow-providers-apache-sqoop/index.rst

Reply:
Example: https://github.com/apache/airflow/tree/master/docs/apache-airflow-providers-odbc